ActindoWorkFlow/Overview/02_WorkflowProcessElements.md

```diff
@@ -0,0 +1,199 @@
+
+A place can hold any number of tokens, including none. A token is a container carrying data, which are needed to execute a subsequent action. The place data type is always defined, more of less specifically, so it is possible to recognize the kind of data that flow through it. For detailed information, see [Tokens](#tokens).
+
+Special types of places are the start and end point. Every workflow starts with a single place and ends with a single place. For detailed information, see [Start and end point](#start-and-end-point).
+
+
+When clicking a place, it becomes highlighted and the corresponding element settings are displayed in the settings side bar.
+
+![Settings side bar](/Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar01.png "[Settings side bar]")
+
+The place settings side bar is located to the right of the workspace and allows to define the place settings. The following fields are displayed:
+
+  - *Key*  
+  Place key. This field is read-only. The key is required for API access and must be system wide unique.
+
+  - *Data container*  
+  Use this field to define the data type to be used in the place. If necessary, click the ![Delete](/Assets/Icons/Cross02.png "[Delete]") (Delete) button to display the whole list of available data types. Enter a search text for the desired data type. The list of actions is filtered for your search text as you type. Select the ![Radio button](/Assets/Icons/Radiobutton.png "[Radio button]") (Radio button) corresponding to the appropriate option.
+
+
+
+## Actions
+
+![Action](/Assets/Screenshots/ActindoWorkFlow/Workflows/AddAction.png "[Action]")
+
+Actions are the individual activities being executed in the workflow and are represented by a rectangle. Actions serve as building blocks, which are chained together through places, to configure a workflow.
+
+All actions contain a certain, predefined number of input and output ports, at least 1 of each, which can be mandatory or optional. For detailed information about the types of input/output ports, see [Input and output ports](#input-and-output-ports). For an action to be executable, all **connected** inputs and outputs, mandatory or optional, must be fed with data.
+
+The actions available to create workflows depend on the modules installed in the system. Every API end point of the modules automatically becomes available as an action and can be used in any workflow. However, you can also build your own actions independent from the API.
+
+[comment]: <> (Ich finde die Benennung API endpoint etwas verwirrend hier, denn wir haben soeben über start/end point als start/end place gesprochen. Alternative Bennenung?)
```

Comment:
Umbenennung von start/end point in start/end place? Dann ist auch direkt klar, dass es sich um einen place handelt...